### PR TITLE
Improves UI stability of booking page([user]/[type]). No CLS now.

### PR DESCRIPTION
**Pull request by hariombalhara**
## What does this PR do?

Fixes # (issue)
Fixes various CLS reasons, See loom https://www.loom.com/share/cc389dc4502d44629b009943689cddf7

The following root causes for CLS are fixed - 
- Profile Images weren't SSRd
- The area where dates appear wasn't rendered initially and thus it was causing left side component to take more space temporarily
- There was black border than can be seen momentarily due to custom CSS properties being added during React useEffect hook which is pretty late. Should have been available on the server itself.
- Preloaded cal-font to make sure text using the font is shown faster.

Also fixed, titles of Avatars not available on
- EventTypes Page
- Teams Page
- Booking Pages

**Environment**: Production

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

File: apps/web/components/CustomBranding.tsx [hunk 1]
```diff
@@ -238,54 +238,32 @@ const BrandColor = ({
...
+      --booking-median-color: ${embedBrandingColors.medianColor || "#494949"};
+      /* gray--800 */
+      --booking-dark-color: ${embedBrandingColors.darkColor || "#313131"};
+      /* gray--900 */
+      --booking-darker-color: ${embedBrandingColors.darkerColor || "#292929"};
+      --brand-color: ${lightVal};
+      --brand-text-color: ${getContrastingTextColor(lightVal, true)};
+      --brand-color-dark-mode: ${darkVal};
+      --brand-text-color-dark-mode: ${getContrastingTextColor(darkVal, true)};
+    `}
```
Comment (zomars) [inline apps/web/components/CustomBranding.tsx:263]: Smart 👍🏽

**Review by zomars — APPROVED** [review]
The Walking Dead Easy Peasy GIF

**Comment by PeerRich** [comment]
btw we call this page "booking page" not availability.

"availability" is /availability

**Review by PeerRich — APPROVED** [review]
awesome job! those tiny improvements will help a lot

**Comment by hariombalhara** [comment]
I try to differentiate b/w [user]/[type] page where you see the availability and /booking page where you actually enter your details as a Booker. 

But that's a valid point, not sure how to avoid the confusion

**Comment by PeerRich** [comment]
huh tests are failing :(

**Review by pumfleet — APPROVED** [review]

**Comment by hariombalhara** [comment]
Failing checks issue sorted and merged @PeerRich

**Pull request merged**